Stack/Opc.Ua.Core/Stack/State/AcknowledgeableConditionState.cs

```diff
@@ -417,6 +423,36 @@ protected virtual void UpdateStateAfterUnconfirm(ISystemContext context)
                 UpdateEffectiveState(context);
             }
         }
+
+        /// <summary>
+        /// Determines if a comment should be added on Acknowledgement or Confirm.
+        /// According to the specification for Alarms, the Acknowledgement states that
```

Comment:
the explaining comment might be better under \<remarks\>?

Reply:
Documentation updated as requested